setup.bash

```diff
@@ -24,13 +24,17 @@ ul () {
 }
 
 sf () {
-    if [[ $@ ]]; then
-        ssh fetch$@
+    if [[ $@ && $FETCH_USER ]]; then
+        ssh $FETCH_USER@fetch$@
+    elif [[ $@ ]]; then
+        ssh fetch@fetch$@
```

Comment:
Should be ssh `fetch$@` which will use the current user by default. This maintains the existing behavior. If you want to set your user to fetch, you can just add `export FETCH_USER=fetch` to your `.bashrc`